libraries/abstractions/mqtt_agent/mqtt_agent_interface.cmake

```diff
@@ -0,0 +1,36 @@
+# MQTT Agent Interface
+afr_module(NAME mqtt_agent_interface INTERNAL)
```

Comment:
As the name of this module (`mqtt_agent_interface`) does not match its parent folder name (i.e. `mqtt_agent`), this CMake files needs to be explicitly added to the metadata list with the following CMake function call.
```
# Add cmake files of module to metadata.
afr_module_cmake_files(${AFR_CURRENT_MODULE} 
    ${CMAKE_CURRENT_LIST_DIR}/mqtt_agent_interface.cmake
)
```

Alternatively, you can rename the parent folder to match the CMake module name to have this file automatically included in the CMake metadata list

Reply:
Isn't that call already present at lines 8-11? I think I'll just keep it as is

Reply:
Oops, I didn't notice it.